api/azure/sdk_type_definition.rb

```diff
@@ -86,6 +86,11 @@ def validate
           check :go_enum_type_name, type: ::String
           check :go_enum_const_prefix, type: ::String, default: ''
         end
+
+        def merge_overrides!(overrides)
```

Comment:
I think here we also need to merge the attributes in `SDKTypeDefinition`
```suggestion
        def merge_overrides!(overrides)
            super
```

Reply:
Oh! My mistake! Thank you for pointing this out!